packages/ui/src/1_atoms/InputBase/InputBase.tsx

```diff
@@ -67,9 +67,7 @@ export const InputBase = React.forwardRef<HTMLInputElement, InputBaseProps>(
 
     // updating value if it was changed by parent component
     useEffect(() => {
-      if (shouldAllowChanges) {
-        setRenderedValue(value as string);
-      }
+      setRenderedValue(value as string);
     }, [shouldAllowChanges, value]);
```

Comment:
```suggestion
    }, [value]);
```

Not sure why this wasn't picked up by linter (running code locally shows no warning either)

Reply:
Good catch, I am also not sure why it hasn't been picked up.